contracts/test/v0.8/automation/KeeperRegistry2_1.test.ts

```diff
@@ -990,41 +990,27 @@ describe('KeeperRegistry2_1', () => {
       .transfer(await admin.getAddress(), toWei('1000'))
     let tx = await registry
       .connect(owner)
-      ['registerUpkeep(address,uint32,address,bytes,bytes)'](
```

Comment:
If this file diff is not relevant to the change in this PR, could we skip it?

Reply:
I see that this is causing lint errors. Resolving this comment

Reply:
for some reason, this was causing a required GH workflow to fail